src/DynamoCore/Services/ShapewaysClient.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Newtonsoft.Json;
+using RestSharp;
+using RestSharp.Contrib;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Dynamo.Services
+{
+    public class ShapewaysClient
```

Comment:
Let's make this `internal` for now. Please add documentation matching the style guide - https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards